src/gsy_e/models/area/area.py

```diff
@@ -91,17 +91,15 @@ def __init__(self, name: str = None, children: List["Area"] = None,
     def get_state(self):
         """Get the current state of the area."""
         state = super().get_state()
-
-        state.update(**{
-            "current_tick": self.current_tick,
-            "area_stats": self.stats.get_state()
-        })
+        state.update({"current_tick": self.current_tick})
```

Comment:
Nothing to be done now, but I think that it is not needed to repeat the `current_tick` for every area, and it should be stored only once. However it is part of a larger refactoring and there is a good chance once we implement it we will design this in a way that avoids repetition of state. TLDR nothing to do now

Reply:
I was also thinking about this. Indeed, lets tackle this in the future.